### PR TITLE
[receiver/awsecscontainermetricsreceiver] Add `ServiceName` metadata to be retrieved from task metadata endpoint (PR)

### PR DESCRIPTION
**Pull request by erichsueh3**
**Description:** <Describe what has changed.>
See tracking issue for description of problem being addressed.

**Link to tracking Issue:** <Issue number if applicable>
#19728 

**Testing:** <Describe what testing was performed and which tests were added.>
`go test -v` in /receivers/awsecscontainermetricsreceiver`

**Documentation:** <Describe the documentation added.>

### CODE REVIEW

File: internal/aws/ecsutil/metadata.go [hunk 1]
```diff
@@ -26,6 +26,7 @@ type TaskMetadata struct {
 	PullStartedAt    string              `json:"PullStartedAt,omitempty"`
 	PullStoppedAt    string              `json:"PullStoppedAt,omitempty"`
 	Revision         string              `json:"Revision,omitempty"`
+	ServiceName			 string							 `json:"ServiceName,omitempty"`
```
Comment (mxiamxia) [inline internal/aws/ecsutil/metadata.go:29]: Pls fix the indentation

Comment (erichsueh3) [reply internal/aws/ecsutil/metadata.go]: Fixed

File: receiver/awsecscontainermetricsreceiver/internal/awsecscontainermetrics/resource.go [hunk 2]
```diff
@@ -63,7 +63,7 @@ func taskResource(tm ecsutil.TaskMetadata) pcommon.Resource {
 	resource.Attributes().PutStr(attributeECSTaskRevision, tm.Revision)
 	resource.Attributes().PutStr(conventions.AttributeAWSECSTaskRevision, tm.Revision)
 
-	resource.Attributes().PutStr(attributeECSServiceName, "undefined")
+	resource.Attributes().PutStr(attributeECSServiceName, tm.ServiceName)
```
[the remark this answers is not in the document]
Comment (erichsueh3) [reply receiver/awsecscontainermetricsreceiver/internal/awsecscontainermetrics/resource.go:66]: Updated

**Review by mxiamxia — APPROVED** [review]
LGTM!

**Review by Aneurysm9 — APPROVED** [review]

**Comment by erichsueh3** [comment]
Hey @codeboten can I get a look at this PR, thanks!

**Review by codeboten — APPROVED** [review]

**Pull request merged**